Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -4,6 +4,9 @@ import AppStoreConnect_Swift_SDK
 import Combine
 import Foundation
 
+import struct Model.App
+import struct Model.BetaGroup
+
```

Comment:
This list will be pretty long, we need network layer 🚁

Reply:
Yeah when we move service / networking moves out we can just use `import Model`